packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -158,6 +162,14 @@ governing permissions and limitations under the License.
   /* allow focus ring of child to extend outside the bounds */
   padding: 4px;
   margin: -4px;
+
+  /*
+   * This is so user provided content follows the column alignment setting.
+   * Has a specificity of 0 1 0, which is the same as a single class.
+   */
+  > * {
+    display: inline;
```

Comment:
This causes the width of the provided div to NOT take 100% of the available width... `display: inline-block` or `display: inline-table` help a little bit, but come with their own issues regarding text truncation. Removing this line and relying on the user to style the div/spans/whatever they pass in has issues with consistent styling and alignment however....